discovery-provider/src/app.py

```diff
@@ -496,6 +497,10 @@ def configure_celery(celery, test_config=None):
                 "task": "index_user_listening_history",
                 "schedule": timedelta(seconds=5),
             },
+            "prune_plays": {
+                "task": "prune_plays",
+                "schedule": timedelta(seconds=21600),  # 4x a day
```

Comment:
Should we use a cron instead of a timedelta? https://docs.celeryproject.org/en/stable/userguide/periodic-tasks.html#crontab-schedules